### PR TITLE
SONARPY-192 Load rule metadata using Rule API

### PR DESCRIPTION
**Pull request by pynicolas**

### CODE REVIEW

File: python-checks/src/main/resources/org/sonar/l10n/py/rules/python/LongIntegerWithLowercaseSuffixUsage.html [hunk 1]
```diff
@@ -1,17 +1,14 @@
...
-</p>
-
-<p>
-The following code snippet illustrates this rule :
-</p>
-
+<p>The long suffix should always be written in uppercase, i.e. 'L', as the lowercase 'l' can easily be confused with the digit one '1'.</p>
+<h2>Noncompliant Code Example</h2>
+<pre>
+long n = 10l;  // Noncompliant; easily confused with one zero one
```
Comment (pynicolas) [inline python-checks/src/main/resources/org/sonar/l10n/py/rules/python/LongIntegerWithLowercaseSuffixUsage.html]: @ivandalbosco This is not Python

File: python-checks/src/main/resources/org/sonar/l10n/py/rules/python/LongIntegerWithLowercaseSuffixUsage.html [hunk 2]
```diff
@@ -1,17 +1,14 @@
...
+</pre>
+<h2>Compliant Solution</h2>
 <pre>
-return 1l  # Non-Compliant
-return 1L  # Compliant
+long n = 10L;
 </pre>
+<h2>See Also</h2>
+<ul>
+  <li> {rule:python:S818} - Literal suffixes should be upper case </li>
```
Comment (pynicolas) [inline python-checks/src/main/resources/org/sonar/l10n/py/rules/python/LongIntegerWithLowercaseSuffixUsage.html]: @ivandalbosco Where does this come from? This link is broken: S818 does not exist for Python.

File: python-checks/src/main/resources/org/sonar/l10n/py/rules/python/S1721.html [hunk 3]
```diff
@@ -1,28 +1,20 @@
...
-
 <pre>
 x = 1
-while x < 10:
+while x &lt; 10:
     print "x is now %d" % (x)
     x += 1
 </pre>
+<h2>Deprecated</h2>
+<p>This rule is deprecated; use {rule:python:S1110} instead.</p>
```
Comment (pynicolas) [inline python-checks/src/main/resources/org/sonar/l10n/py/rules/python/S1721.html]: @ivandalbosco I think this rule should not be deprecated anymore: I added a comment on https://jira.sonarsource.com/browse/RSPEC-1110

File: python-checks/src/main/resources/org/sonar/l10n/py/rules/python/LongIntegerWithLowercaseSuffixUsage.html (hunk 1 above)
Comment (ivandalbosco) [inline python-checks/src/main/resources/org/sonar/l10n/py/rules/python/LongIntegerWithLowercaseSuffixUsage.html]: @pynicolas fixed in https://jira.sonarsource.com/browse/RSPEC-1129

File: python-checks/src/main/resources/org/sonar/l10n/py/rules/python/LongIntegerWithLowercaseSuffixUsage.html (hunk 2 above)
Comment (ivandalbosco) [inline python-checks/src/main/resources/org/sonar/l10n/py/rules/python/LongIntegerWithLowercaseSuffixUsage.html]: @pynicolas Whole "See also" section removed from RSPEC. I didn't create any ticket to implement S818, as not everything in S818 is relevant for Pyton. For example, hex format can be like '02x' or '02X' for good reasons (to generate "ff" vs. "FF" for 255).

File: python-checks/src/main/resources/org/sonar/l10n/py/rules/python/S1721.html (hunk 3 above)
Comment (ivandalbosco) [inline python-checks/src/main/resources/org/sonar/l10n/py/rules/python/S1721.html]: @pynicolas OK, waiting for Ann's response.

**Pull request merged**